### DISCUSSION TITLE
what is the mean of omc_CodegenC_fun__xxx?

### DISCUSSION THREAD

**Discussion by pily1** [OpenModelica/OpenModelica#12412] [Q&A] [answered]
there are many functions like "omc_CodegenC_fun_xxx" in CodegenC.c file,what's the mean of the xxx?
what's the difference between "omc_CodegenC_fun_xxx" with "omc_CodegenC_fun_yyy"

**Comment by perost** [comment, marked as answer]
There's no particular meaning, it's just how C code is generated from Susan where a template might need to be split into multiple C functions.
